src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/ImportOrchestratorJob.cs

```diff
@@ -285,12 +285,12 @@ private async Task SendImportMetricsNotification(JobStatus jobStatus, JobInfo jo
 
                 _auditLogger.LogAudit(
                 AuditAction.Executed,
-                operation: ImportMode.IncrementalLoad.ToString(),
+                operation: "import/" + ImportMode.IncrementalLoad.ToString(),
```

Comment:
I am not sure of the requirement, but just want to make sure if it should include $ symbol as well?
Also, probably we can use constant
https://github.com/microsoft/fhir-server/blob/76a66019b247fdfc587f1152e0eb931fc069110e/src/Microsoft.Health.Fhir.Api/Features/Routing/KnownRoutes.cs#L65